apps/web/src/types/routes.ts

```diff
@@ -47,3 +47,5 @@ export enum RoutePatterns {
   ResetPassword = "(/)reset-password(/)",
   VerifyEmail = "(/)verify-email(/)",
 }
+
+export const CALLBACK_URL = window.location.origin.concat(Route.VerifyEmail);
```

Comment:
- We already have this https://github.com/litespace-org/litespace/blob/51205b4a3698bc35e527b049c94d6bfb1f7f4aeb/apps/web/src/types/routes.ts#L33-L35
- Also the name should be specific because we have another callback url for forgetting the password.